contracts/truefi2/interface/ILoanToken2.sol

```diff
@@ -5,43 +5,38 @@ import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {ERC20} from "../../common/UpgradeableERC20.sol";
 import {ITrueFiPool2} from "./ITrueFiPool2.sol";
 
-// These two interfaces have to stay in a single file due to a flattener bug that causes IDebtToken to be declared before ILoanToken2
-interface IDebtToken is IERC20 {
+interface ILoanToken2 is IERC20 {
     enum Status {Awaiting, Funded, Withdrawn, Settled, Defaulted, Liquidated}
```

Comment:
I think we should break backward compatibility entirely and remove all Statuses except `{Withdrawn, Settled, Defaulted}` for ILoanToken3.